pkg/disttask/framework/testutil/context.go

```diff
@@ -354,7 +357,7 @@ func (c *TestDXFContext) electIfNeeded() {
 	newOwnerIdx := int(rand.Int31n(int32(len(c.mu.nodes))))
 	ownerNode := c.mu.nodes[newOwnerIdx]
 	c.mu.ownerIndices[ownerNode.id] = newOwnerIdx
-	ownerNode.schMgr = scheduler.NewManager(c.Ctx, c.TaskMgr, ownerNode.id)
+	ownerNode.schMgr = scheduler.NewManager(c.Ctx, c.TaskMgr, ownerNode.id, proto.NodeResourceForTest)
```

Comment:
also need change to `proto.NewNodeResource(c.mockCPUNum, 32*units.GB, 100*units.GB)`